epay3.Web.Api.Tests/App.config

```diff
@@ -1,28 +1,28 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Were these changes necessary for the AutoPay create ticket? Or did you commit these changes unintentionally....?